### ISSUE TITLE
[Feature Request] Quickly return to homepage 

### ISSUE THREAD

**Issue by Fmoment1** [Emplexx/Aluminium#40]
Helpful for when you've gone too deep into anime/manga relations and want to go back to the homepage quickly. Currently we have to manually go back each page until we reach square one.

**Comment by Emplexx** [comment]
Unfortunately I don't think this will happen due to technical limitations. The app is using an activity (think of it like a window) per screen, while having bottom navigation on all screens would require for all screens to be fragments (which are embeddable into activities) and the app to use a single-activity architecture. 

Basically I'd need to rewrite all screens while also losing benefits such as hassle free navigation, (good) predictive back animations, easy to work with screens (fragments are infamous for being weird), etc

**Comment by Fmoment1** [comment]
Understandable. Scrapping this idea, could it be possible to implement a way for users to go back to homepage in one action, within the current limitations?

**Comment by Emplexx** [comment]
I can only think of long pressing the back button for it or something like that

**Comment by Fmoment1** [comment]
That sounds nice, would be better if it's something more attuned to one-handed usage, but I can't complain too much. I'm changing the title to be more appropriate. 

This isn't really a big deal for me btw, just a feature that'd be "nice to have". Thanks for considering!

**Comment by 0-BlackSpectrum-0** [comment]
> Unfortunately I don't think this will happen due to technical limitations. The app is using an activity (think of it like a window) per screen, while having bottom navigation on all screens would require for all screens to be fragments (which are embeddable into activities) and the app to use a single-activity architecture. 
> 
> Basically I'd need to rewrite all screens while also losing benefits such as hassle free navigation, (good) predictive back animations, easy to work with screens (fragments are infamous for being weird), etc 

How about persistent nav bar that would stay there at the bottom in every page, does current limitation not allow that? (you can make the nav bar hide when scrolling)
Just an idea it'd be nice if this works!

**Comment by Emplexx** [comment]
> How about persistent nav bar that would stay there at the bottom in every page, does current limitation not allow that? (you can make the nav bar hide when scrolling)
> Just an idea it'd be nice if this works!

No, that is pretty much what I described as being impossible with the current app architecture.
